src/components/Ui/Markdown.vue

```diff
@@ -19,6 +22,26 @@ const markdown = computed(() => {
   // body = sanitizeHtml(body);
   return body;
 });
+
+onMounted(() => {
+  document
+    .querySelector('.markdown-body')
+    .querySelectorAll('pre>code')
+    .forEach(function (code) {
```

Comment:
Good opportunity to add the missing border radius to the pre element.

```suggestion
    .forEach(function (code) {
      code.parentElement.classList.add('rounded-lg');
```